README.md

```diff
@@ -1,9 +1,19 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+<!-- For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news). -->
 
+If you have cloned this repo to your local, you will need to complete a couple of setup tasks. Please see below:
 
+1. Create two .env files: .env.test and .env.development.
+2. Within these .env files, you will need to add PGDATABASE= with the correct database name for that environment.
 
---- 
+- .env.test => PGDATABASE=nc_news_test
+  -.env.development => PGDATABASE=nc-news
+
+3. Create a .gitignore file if you haven't done so already. Within these, type node_modules and .env.\* (or as separate files .env.test & .env.development) which will insure that these files are ignored by GitHub.
```

Comment:
This step shouldnt be here. As you have updated the .gitignore file to include this it should be included for everyone who clones this down so it shouldnt need to be done again.